### ISSUE TITLE
[Android] Create shortcuts

### ISSUE THREAD

**Issue by AJenbo** [diasurgical/DevilutionX#3197]
It would be good to provide the ability to create shortcuts for starting the game in either Diablo or Hellfire mode directly. This should be doable using the following documentation:

https://developer.android.com/guide/topics/ui/shortcuts/creating-shortcuts

Additionally there should be a short cut for accessing the DataActivity so that to allow users to fetch additional language packages after installation.

**Comment by bubio** [comment]
It would be useful for iOS to support QuickAction as well.

https://developer.apple.com/design/human-interface-guidelines/ios/system-capabilities/home-screen-actions/

**Comment by AJenbo** [comment]
Yeah that sounds like a pretty close concept, though it should probably have it's own task as I'm thinking the actual implementations cannot be shared so most likely would be solved and work on separately.
